shared/login/register/paper-key/index.desktop.js

```diff
@@ -47,7 +47,7 @@ const styles = {
   },
   input: {
     marginBottom: 55,
-    height: '4em',
+    height: '5em',
```

Comment:
When testing provisioning and entering in paper key, my paper key was wrapping and causing scrollbar, bumping up the height, makes the input nicer and can see all of paper key entry.